hnn_core/gui/gui.py

```diff
@@ -26,15 +26,307 @@
                              _read_legacy_params)
 from hnn_core.viz import plot_dipole
 
-THEMECOLOR = "#8A2BE2"
 
+class HNNGUI:
+    """HNN GUI class
+
+    Parameters
+    ----------
+    theme_color : str
+        The theme color of the whole dashboard.
+    log_window_height: str
+        The height of the log output window (in pixel).
+    visualization_window_width: str
+        The width of the visualization window (in pixel).
+    visualization_window_height: str
+        The height of the visualization window (in pixel).
+    left_sidebar_width: str
+        The width of the left side bar (in pixel).
+    TODO: add more parameters.
```

Comment:
I love the idea of having access to these parameters in the constructor rather than just hard-coding them, but I'm not clear on how/when they'll be used. Out of curiosity, what's your high-level vision here @chenghuzi?

Reply:
You can check that now in our latest gui tests